tests/integration/base.py

```diff
@@ -150,15 +151,40 @@ def databricks_cluster_profile(self):
             },
             'test': {
                 'outputs': {
-                    'dbsql': {
+                    'cluster': {
                         'type': 'databricks',
                         'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
                         'http_path': os.getenv('DBT_DATABRICKS_CLUSTER_HTTP_PATH'),
                         'token': os.getenv('DBT_DATABRICKS_TOKEN'),
                         'schema': self.unique_schema()
                     },
                 },
-                'target': 'dbsql'
+                'target': 'cluster'
+            }
+        }
+
+    def databricks_uc_cluster_profile(self):
```

Comment:
I wonder if we can combine the UC cluster profile with the regular cluster profile by simply giving it a cluster HTTP path that points it to a UC-enabled cluster during the integration test (with the initial catalog set to different values).

Reply:
I'm afraid I don't think we should do that.
The UC cluster and non-UC cluster have some different behaviors, e.g., the UC cluster doesn't allow to write non-delta tables.
Having another profile makes switching tests easier.

Reply:
Thanks for the explanation. My intention is to reuse the code and the environment variable a bit more here. For example, if in the future we want to test the behavior when ANSI is enabled, then we might need to add more profiles. I am thinking if we can do something like this:
```python
def build_databricks_profile(self, session_properties: Dict[str, str] = None):
      profile = {
          'config': {
              'send_anonymous_usage_stats': False
          },
          'test': {
              'outputs': {
                  'default2': {
                      'type': 'databricks',
                      'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
                      'http_path': os.getenv('DBT_DATABRICKS_HTTP_PATH'),  # this can be cluster or SQL endpoint HTTP path
                      'token': os.getenv('DBT_DATABRICKS_TOKEN'),
                      'schema': self.unique_schema()
                  },
              },
              'target': 'default2'
          }
      }
     if session_properties is not None:
        ... 
     return profile

def databricks_uc_cluster_profile(self):
    return self.build_databricks_profile(session_properties=dict(...))
```
Or we can also pass in `http_path` as a param.

Reply:
Refactoring like `build_databricks_profile` sounds good.
I'll modify that.

However, reusing the environment `DBT_DATABRICKS_HTTP_PATH` is not a good idea, IMO.

Thinking of run the integration tests in the local environment manually, we will have to modify the environment variable as well as the profile name.

For example, we want to run `integration-databricks-cluster` and `integration-databricks-uc-cluster`, then:

```
export DBT_DATABRICKS_HTTP_PATH=<http-path for non-uc cluster>
tox -e integration-databricks-cluster
export DBT_DATABRICKS_HTTP_PATH=<http-path for uc cluster>
tox -e integration-databricks-uc-cluster
```

I'd prefer to set the environment variables in advance like in a profile file, then run the tox command in a row:

in a profile file:

```
export DBT_DATABRICKS_CLUSTER_HTTP_PATH=<http-path for non-uc cluster>
export DBT_DATABRICKS_UC_CLUSTER_HTTP_PATH=<http-path for uc cluster>
```

then run tox commands:

```
tox -e integration-databricks-cluster
tox -e integration-databricks-uc-cluster
```

We don't need to take care of the environment variables anymore.

Reply:
Ah, reusing `DBT_DATABRICKS_HTTP_PATH` for delegation should be fine.
e.g., `os.getenv('DBT_DATABRICKS_CLUSTER_HTTP_PATH', os.getenv('DBT_DATABRICKS_HTTP_PATH'))`
Actually I'm going to do that at #51.
We can do that here, too.

Reply:
Let me apply the changes here.